packages/atomic/src/components/facets/atomic-timeframe-facet/atomic-timeframe-facet.tsx

```diff
@@ -154,10 +154,15 @@ export class AtomicTimeframeFacet
 
   private get currentValues(): DateRangeRequest[] {
     return this.manualTimeframes.map(({period, amount, unit}) =>
-      buildDateRange({
-        start: {period, unit, amount},
-        end: {period: 'now'},
-      })
+      period === 'past'
```

Comment:
Since `period` can be `past | now | next` should we handle all three in a switch case ? 

Event though having `now` configured here would be... illogical...

Reply:
You can't have now, it's not an option on the atomic-timeframe component